### PR TITLE
fix(client): remove `staleTime`

### PR DESCRIPTION
**Pull request by lluia**
## Summary 💭

Following the removal of `staleTime` from the **client module** on `next`, we had to remove references to it in the docs.

Given `staleTime` is already implicitly set when setting `refetchInterval`, do you think we need to reword the `refetchInterval` docs a bit to indicate that? ✍🏼

### CODE REVIEW

**Review by balazsorban44 — COMMENTED** [review]
I am not sure I like the wording "implicit" here. It is not *exactly* the same, and it might only be me, but to me, it sounds like that is what we are saying. Maybe rephrase to something like 

`clientMaxAge` has been removed in favor of `refetchInterval`, which overlaps with the functionality, with the difference that it will keep refetching the session periodically in the background.

Something like this at least.

**Review by ndom91 — APPROVED** [review]
Nice! LGTM :+1:

**Review by balazsorban44 — APPROVED** [review]
